Public/Sdk/Public/Managed/Testing/XUnit/xunit.dsc

```diff
@@ -65,6 +65,14 @@ export function runConsoleTest(args: TestRunArguments): Result {
 
     const result = Transformer.execute(execArguments);
 
+    if (Environment.hasVariable("BUILDXL_IS_IN_CLOUDBUILD")) {
+        const qualifierRelative = r`${qualifier.configuration}/${qualifier.targetFramework}/${qualifier.targetRuntime}`;
+        const parallelRelative = args.parallelBucketIndex !== undefined ? `${args.parallelBucketIndex}` : `0`;
+        const xunitLogDir = d`${Context.getMount("LogsDirectory").path}/XUnit/${Context.getLastActiveUseModuleName()}/${Context.getLastActiveUseName()}/${qualifierRelative}/${parallelRelative}`;
+
+        const copies = result.getOutputFiles().map(f => Transformer.copyFile(f, p`${xunitLogDir}/${f.name}`));

```

Comment:
copies are not used later on. Do you need to save it?

Reply:
I don't need it; potentially a left over from my experiments.

---
In reply to: [293204603](https://github.com/microsoft/BuildXL/pull/457#discussion_r293204603) [](ancestors = 293204603)